### PR TITLE
Automate manifest saas metering

### PR DESCRIPTION
**Pull request by EmilyM1**

### CODE REVIEW

File: hack/create-metering-manifests.sh [hunk 1]
```diff
@@ -22,6 +22,10 @@ if [[ $# -ge 3 ]] ; then
     mkdir -p "${OLM_OUTPUT_DIR}"
     shift
 
+    TELEMETER_OUTPUT_DIR=$1
+    echo "Telemeter manifest output directory: ${TELEMETER_OUTPUT_DIR}"
+    mkdir -p "${TELEMETER_OUTPUT_DIR}"
```
Comment (chancez) [inline hack/create-metering-manifests.sh]: need to add a `shift` after this line to shift arguments.

File: manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -206,7 +206,7 @@ metadata:
     categories: OpenShift Optional, Monitoring
     certified: "false"
     containerImage: quay.io/openshift/origin-metering-ansible-operator:latest
-    createdAt: 2019-01-01T11:59:59Z
+    createdAt: "2019-01-01T11:59:59Z"
```
Comment (timflannagan1) [inline manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: Remove these quotation marks, I believe this is what is causing verify to fail.

File: manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 3]
```diff
@@ -538,4 +538,3 @@ spec:
       kind: HiveTable
       name: hivetables.metering.openshift.io
       version: v1
-
```
Comment (timflannagan1) [inline manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: Add this line back and verify should pass

Comment (timflannagan1) [reply manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: Remove the indentation from this line - you can configure your editor to do this by default (removing spaces from an empty line)

File: hack/generate-metering-manifests.sh [hunk 4]
```diff
@@ -9,4 +9,5 @@ trap "rm -rf $TMPDIR" EXIT
 
 "$ROOT_DIR/hack/create-metering-manifests.sh" \
     "$INSTALLER_MANIFESTS_DIR" \
-    "$OLM_MANIFESTS_DIR"
+    "$OLM_MANIFESTS_DIR" \
+    "$TELEMETER_MANIFESTS_DIR" \
```
Comment (chancez) [inline hack/generate-metering-manifests.sh]: trailing `\` needs to be removed here

File: manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml [hunk 5]
```diff
@@ -538,4 +538,4 @@ spec:
...
+
```
Comment (chancez) [inline manifests/deploy/openshift/olm/bundle/4.2/meteringoperator.v4.2.0.clusterserviceversion.yaml]: Trailing whitespace shouldnt be here

**Comment by chancez** [comment]
e2e is failing because we use `create-metering-manifests.sh` in one of our e2e scripts, and you need to update it to pass the new arg.

**Pull request merged**